### DISCUSSION TITLE
Using ROBOT templates to bulk add new terms or annotations to terms

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [jamesaoverton/obook#153] [Tutorial requests]
For example, I need to add new synonyms that were provided in a table from an external collaborator to existing terms.

Scenario:
I get a big table of synonyms to add to existing terms in my ontology, consider provenance, source, etc. Demonstrate how to merge into the ontology.

**Comment by nicolevasilevsky** [comment]
I know how to do this in Mondo but not in other ontologies, so this is something I'd like to know.

**Comment by jamesaoverton** [comment]
This documentation on one-off ROBOT templates might be relevant: https://github.com/EnvironmentOntology/envo/wiki/ENVO-Robot-template-and-merge-workflow
